docs/commands.md

```diff
@@ -6427,6 +6427,7 @@ hal config provider azure account add ACCOUNT [parameters]
  * `--required-group-membership`: (*Default*: `[]`) A user must be a member of at least one specified group in order to make changes to this account's cloud resources.
  * `--subscription-id`: (*Required*) The subscriptionId that your service principal is assigned to.
  * `--tenant-id`: (*Required*) The tenantId that your service principal is assigned to.
+ * `--useSshPublicKey`: (*Default*: `true`) Whether to use SSH public key to provision the linux vm. The default value is true which means using the ssh public key. Setting it to false means using the password instead.
```

Comment:
It would be worth specifying that the actual SSH public key is taken from the KeyVault pointed by the parameter `--default-key-vault`, and it should be a secret named `VMSshPublicKey`. Similarly, documentation for `--default-key-vault` should specify that the username/password should be secrets called `VMUsername` and `VMPassword` in that vault.

Reply:
Thank you german for your comments! Specifying how these parameters are used besides what they are is definitely needed. May I suggest we put those detail, which may consist of several lines of texts, in online documents instead of "parameter-auto-description"? As for the latter location we would like to clarify what the parameter is and what it serves in brief words.
On the other hand, I admit now that documents for this are not complete yet. Updating documents is in our short term plan and we will definitely involve your suggested usage instruction here into those documents.

Reply:
Describing full details of parameter usage in separate online docs is fine. I'd suggest then to just update the description of `--default-key-vault` to indicate that it also holds the public SSH key.

Reply:
Make sense. Docs updated.